src/core/qgsvectordataprovider.h

```diff
@@ -88,6 +88,7 @@ class CORE_EXPORT QgsVectorDataProvider : public QgsDataProvider, public QgsFeat
                                                                *  ChangeGeometries | ChangeAttributeValues */
       RenameAttributes = 1 << 19, //!< Supports renaming attributes (fields). Since QGIS 2.16
       FastTruncate = 1 << 20, //!< Supports fast truncation of the layer (removing all features). Since QGIS 3.0
+      CancelSupport = 1 << 21, //!< Supports interruption of pending queries from a separated thread. Since QGIS 3.0
```

Comment:
3.0 or 3.2?

Reply:
@DelazJ Good point, it's 3.2! I'll update the documentation.